### PR TITLE
i/builtin: allow to lock /run/netns with network-control

### PR DESCRIPTION
**Pull request by pedronis**
LP: #2047798

we have similar permissions for /etc/resolvconf

### CODE REVIEW

File: interfaces/builtin/network_control.go [hunk 1]
```diff
@@ -286,7 +286,7 @@ capability sys_admin, # for setns()
 network netlink raw,
 
 / r,
-/run/netns/ r,     # only 'r' since snap-confine will create this for us
+/run/netns/ rk,     # no 'w' since snap-confine will create this for us
```
Comment (bboozzoo) [inline interfaces/builtin/network_control.go:289]: For reference, this is in line with how `ip netns` works https://github.com/iproute2/iproute2/blob/f22c49730c3691c25a1147081363eb35aa9d1048/ip/ipnetns.c#L819

**Review by bboozzoo — APPROVED** [review]
LGTM

**Review by MiguelPires — APPROVED** [review]
lgtm

**Review by alexmurray — APPROVED** [review]
LGTM - thanks for the link to the upstream code as well.

**Pull request merged**